roles/bootstrap-os/tasks/main.yml

```diff
@@ -11,6 +11,10 @@
 - include: bootstrap-centos.yml
   when: bootstrap_os == "centos"
 
+- task:
+  debug:
+    msg: "bootstrap_os: {{ bootstrap_os|d('') }}"
```

Comment:
remove this before merge

Reply:
Did you mean to remove this?

Reply:
I did! I'll take care of it in a follow up. Thanks @jduhamel